test/unit/models_test.rb

```diff
@@ -49,7 +49,7 @@ class ModelsTest < ActiveSupport::TestCase
       'ProxyConfig' => %w[hosts],
       'ProxyRule' => %w[metric_system_name owner_type],
       'ServicePlan' => %w[issuer_type],
-      'Settings' => Switches::SWITCHES.map { |switch| "#{switch}_switch" },
+      'Settings' => [*Switches::SWITCHES.map { |switch| "#{switch}_switch" }, "cms_token"],
```

Comment:
I somehow find it more readable like this. Not a big concern though.
```suggestion
      'Settings' => Switches::SWITCHES.map { |switch| "#{switch}_switch" } << "cms_token"
```

Reply:
https://github.com/3scale/porta/pull/4075/commits/ef888e97c3ae7a0363f3a8647922faee243ba01c